charts/k8ssandra/templates/prometheus/service_monitor.yaml

```diff
@@ -1,8 +1,10 @@
+{{- if .Values.monitoring.prometheus.provision_service_monitors }}
```

Comment:
I was thinking that maybe we should also check that prometheus-operator is enabled to ensure that the ServiceMonitor CRD is installed. Going forward though, the user could install prometheus-operator in another release or install it separately.

Maybe we use the `lookup` function here to ensure that the ServiceMonitor CRD is installed. Thoughts?

Reply:
Good point on the `lookup` function to see if the CRD is present. I would use it in conjunction with this check.